io.github.jakubmelka.pdf4qt.json

```diff
@@ -0,0 +1,256 @@
+{
+	"id": "io.github.jakubmelka.pdf4qt",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.15-21.08",
+	"sdk": "org.kde.Sdk",
+	"command": "pdf4qt",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland"
+	],
+	"modules": [{
```

Comment:
```suggestion
	"modules": [
	    {
```

Reply:
Sorry about that, it's kinda too late to stop the build, but if it fails again I'll change the braces.

Reply:
it doesn't impact the build, it just impact how readable the JSON is, which is why you had issues before.